package.json

```diff
@@ -22,11 +22,11 @@
     }
   ],
   "scripts": {
-    "size": "size-limit",
-    "test": "standard && ava",
-    "testci": "standard && ava && npm run size",
-    "prepublish": "npm run check-deps",
-    "check-deps": "nsp check"
+     "size": "size-limit",
```

Comment:
The indentation is off by 1 here, which makes the diff harder to read. can you leave it as the old indentation?